"The system cannot find the path specified." on time_zone check in pagerduty_schedule

Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
```
Terraform v1.1.8
on windows_amd64
```
Provider: `pagerduty/pagerduty@v2.4.0`
### Affected Resource(s)
- pagerduty_schedule

### Terraform Configuration Files
```hcl
variable "team_name" {
  default = "test-terraform"
}

variable "layers" {
  default = {
    main = {
      start         = "2022-04-18T20:00:00-05:00"
      virtual_start = "2022-04-18T20:00:00-05:00"
      duration      = 3600
    }
  }
}

resource "pagerduty_team" "team" {
  name   = var.team_name
}

resource "pagerduty_schedule" "schedule" {
  name      = var.schedule_name
  time_zone = "America/Chicago"
  teams     = [pagerduty_team.team.id]

  dynamic "layer" {
    for_each = var.layers
    content {
      name  = layer.key
      start = layer.value.start

      rotation_virtual_start       = layer.value.virtual_start
      rotation_turn_length_seconds = layer.value.duration
      users                        = [pagerduty_team.team.id]
    }
  }
}
```

### Expected Behavior
Terraform plan and apply run successfully, pagerduty team and schedule created.

### Actual Behavior
Pagerduty team created successfully. Error on pagerduty schedule
```
│ Error: The system cannot find the path specified.
│
│   with pagerduty.pagerduty_schedule.schedule,
│   on escalation-policy\schedule.tf line 3, in resource "pagerduty_schedule" "schedule":
│    3:   time_zone = var.time_zone
```


### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform plan` or `terraform apply`

### Important Factoids
- Error only occurs on Pagerduty provider 2.4.0. Issue does not appear in 2.3.0 or earlier.
- Error occurs for any input into `time_zone` ("CST", "foo", "bar", "America/New York")
- Changes made in pagerduty/pagerduty@v2.4.0 include validation using `time.LoadLocation`
- Cursory search of this error related to `time.LoadLocation` in GoLang shows timezone check failures on Go 1.13 (https://stackoverflow.com/questions/59044243/timezones-failing-to-load-in-go-1-13) not sure if this is related or not
- Different error (but I believe to be similar root cause) inside docker desktop using wsl backend:

```
$ docker-compose -f docker-compose.yml run --rm terraform version
Creating pagerduty_terraform_run ... done
Terraform v1.1.8
on linux_amd64
+ provider registry.terraform.io/pagerduty/pagerduty v2.4.0
...
$ docker-compose -f docker-compose.yml run --rm terraform plan -var-file=variables.tfvars
Creating pagerduty_terraform_run ... done
╷
│ Error: unknown time zone America/Chicago
│
│   with module.pagerduty.pagerduty_schedule.schedule,
│   on escalation-policy/schedule.tf line 3, in resource "pagerduty_schedule" "schedule":
│    3:   time_zone = "America/Chicago"
```
- Again, error is resolved by rolling back to pagerduty 2.3.0:
```
 docker-compose -f docker-compose.yml run --rm terraform version
Creating pagerduty_terraform_run ... done
Terraform v1.1.8
on linux_amd64
+ provider registry.terraform.io/pagerduty/pagerduty v2.3.0
...
$ docker-compose -f docker-compose.yml run --rm terraform plan -var-file=variables.tfvars
Creating pagerduty_terraform_run ... done

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # module.pagerduty.pagerduty_schedule.schedule will be created
  + resource "pagerduty_schedule" "schedule" {
      + description = "Managed by Terraform"
      + id          = (known after apply)
      + name        = "Terraform"
      + teams       = (known after apply)
      + time_zone   = "America/New York"

      + layer {
          + id                           = (known after apply)
          + name                         = "main"
          + rotation_turn_length_seconds = 3600
          + rotation_virtual_start       = "2022-04-18T20:00:00-05:00"
          + start                        = "2022-04-18T20:00:00-05:00"
          + users                        = (known after apply)
        }
    }
```
### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- possibly [golang/go-50248](https://github.com/golang/go/issues/50248)

I have the same issue here, migrate from 1.11.0 to 2.4.1 

```bash
2022-05-02T13:28:16.692Z [ERROR] vertex "pagerduty_schedule.nightshift" error: unknown time zone Europe/Berlin

Error: unknown time zone Europe/Berlin
```

using the example from official documentation: https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs/resources/schedule#time_zone

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -6,6 +6,8 @@ builds:
     - -trimpath
   ldflags:
     - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
+  tags:
+    - timetzdata
   goos:
     - freebsd
     - windows

